.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
can you remove this from the branch personal changes like this shouldn't be added for everyone.

add the .vscode folder to the gitignore file.